### DISCUSSION TITLE
ERGMS - How to include triads?

### DISCUSSION THREAD

**Discussion by ElenaOS97** [statnet/ergm#451] [Q&A]
1. **GWESP**

I am reading contradictory definitions of gwesp:

a) number of transitive triangles

b) number of k-triangles (transitive triangles clustered)

c) the robust equivalent to the option triangle (which is used for both transitive and cyclic triangles).    

 I wonder which of those is true. Does gwesp measure transitive triangles or does it also measure cyclic triangles?         

2.  **DGWDSP ((type="OTP")  and (type="ITP")**

Does dgwdsp (type="ITP")  measure cyclical clustered triangles​; while dgwdsp (type="OTP")  measures transitive clustered triangles? 

3. **GWDSP**

Does this measure multiple 2-paths?

**Comment by martinamorris** [comment]
Hi Elena,

It would help if you would provide a link to where you are getting your definitions from -- is this from our software or somewhere else?  

The DGW* and GW* terms are not simple counts of any type of triad, they are more complicated weighted sums, where the weights reflect declining marginal returns for each additional "shared partner" (the SP in GWESP).

Have you read this paper:  https://journals.sagepub.com/doi/10.1111/j.1467-9531.2006.00176.x ?  Are you familiar with the difference between counts of triangles (or directed triad configurations) and the "new specifications" that were developed to address the drawbacks of those simple counts in modeling?

**Reply by ElenaOS97** [reply]
Hi Martina, 
I am always equalling the decay parameter to 0, so I am not using the weights.  My question is more whether:

a) Is gwesp  a new configuration of "triangle", "ctriple" or "ttriple"?
b) And what about dgwdsp and gwdsp?

I would like to model the number of clustered transitive triangles, clustered cyclical triangles and clustered independent 2-paths. 
 
In page  43 of the book by Lusher and Robins, they compute an ERGM including parameters for transitivity, cyclic closure and multiple 2-paths. I am aiming to do the same. 
The link for the book is:
https://www.cambridge.org/core/books/exponential-random-graph-models-for-social-networks/9296EE2B53CDEF9FE9E2E981E2FDB8A8
